plugin/pkg/admission/eventratelimit/apis/eventratelimit/types.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package eventratelimit
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// Type of limit (e.g., per-namespace)
+type LimitType string
+
+const (
+	// ServerLimitType limits are maintained against all events received by the server
+	ServerLimitType LimitType = "server"
```

Comment:
All constants in Kubernetes APIs are `CamelCase`.  For the last item, use `SourceAndObject` or similar.